test/Azure.Functions.Cli.Tests/E2E/StartTests.cs

```diff
@@ -32,13 +32,13 @@ await CliTester.Run(new RunConfiguration
                 {
                     "init . --worker-runtime node",
                     "new --template \"Http trigger\" --name HttpTrigger",
-                    "start"
+                    "start --port 7077"
```

Comment:
Can you add some information describing why the port is now being passed in?

Reply:
Removed this; was a previous change